back-end/nats-mail-notifier/tests/nats_service_test.bal

```diff
@@ -0,0 +1,68 @@
+import ballerina/file;
+import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/test;
+import ballerinax/googleapis.gmail;
+import ballerinax/nats;
+
+final nats:Client natsClient = check new (natsUrl);
+final string logFilePath = check file:createTemp(".log", "test-");
+
+public client class MockGmailClient {
+    resource isolated function post users/[string userId]/messages/send(gmail:MessageRequest request,
+            gmail:Xgafv? xgafv = (), string? access_token = (), gmail:Alt? alt = (), string? callback = (),
+            string? fields = (), string? 'key = (), string? oauth_token = (), boolean? prettyPrint = (),
+            string? quotaUser = (), string? upload_protocol = (), string? uploadType = ()) returns gmail:Message|error {
+        string[]? to = request?.to;
+        if to is () {
+            return error("mail to: is empty");
+        }
+        if to.length() != 1 {
+            return error("mail to: is empty");
```

Comment:
We can do this right?
```bal
if to is () || to.length() != 1 {
        return error("mail to: not found");
}
```

Reply:
Fixed with 8c96eb98d493696704a5f3b28e6ef88edea9b765